asciidoc-confluence-publisher-client/src/main/java/org/sahli/asciidoc/confluence/publisher/client/http/ConfluenceRestClient.java

```diff
@@ -171,6 +182,14 @@ public void updateAttachmentContent(String contentId, String attachmentId, Input
             return null;
         });
     }
+    @Override
+    public void updateAttachmentContent(String contentId, String attachmentId, Path attachmentContent, boolean notifyWatchers) {
+        HttpPost updateAttachmentContentRequest = this.httpRequestFactory.updateAttachmentContentRequest(contentId, attachmentId, attachmentContent, notifyWatchers);
+        sendRequestAndFailIfNot20x(updateAttachmentContentRequest, (response) -> {
+            //closeInputStream(attachmentContent);
```

Comment:
Suggestion: Commented out code should either be removed, or it should come with an explanatory comment.